devtools/platform-descriptor-json/src/main/resources/codestarts/quarkus/core/examples/commandmode-example/java/src/main/java/org/acme/{main.class-name}.tpl.qute.java

```diff
@@ -1,4 +1,4 @@
-package {package-name};
+package org.acme;
```

Comment:
Does this mean it's not a user's choice anymore?

Reply:
@aloubyansky nop, just it's now handled the same way for all codestarts `org.acme` is replaced by the selected package name at gen time.

Reply:
In that case why not keep the parameter with the default value `org.acme`?

Reply:
Those "core" examples (resteasy, springweb and commandmode) were a bit special and allowed change in the class name and package name.

To avoid unnecessary templating in most codestarts like this one:
https://github.com/quarkusio/quarkus/blob/master/devtools/platform-descriptor-json/src/main/resources/codestarts/quarkus/examples/funqy-http-example/java/src/main/java/org/acme/funqy/Funqy.java

We decided to make it a convention to use `org.acme` and auto replace it for all java/kotlin/scala files:
https://github.com/quarkusio/quarkus/blob/master/independent-projects/tools/codestarts/src/main/java/io/quarkus/devtools/codestarts/core/strategy/SmartPackageFileStrategyHandler.java

I kept that here thinking it was ok, but then if you use `org.acme.something`, it gets replaced twice leading to the actual bug :)

Reply:
dynamic package should not be done with `{package-name}` anymore, or else, we should either:
- make a change so that we don't do the replace twice which is can of hard to know since we would have mixed approach
- make sure the dynamic package doesn't contain `org.acme.*`

but I don't see the point since the current `org.acme` is consistent and you don't have to care about it..

Reply:
It looks like the package is generally modeled as `{root-package}[.{extension-name}]` It would be more intuitive if it was configured as a kind of expression.
Using a specific constant `org.acme` as the default and a placeholder is confusing and not practical in case one day you want to change the default value to something else.
That's just my opinion.

Reply:
@aloubyansky  We could still easily change the default, it just means the codestart template would keep `org.acme` which is fine (or else we just have to refactor).

The big advantage of that convention is that most example classes which don't use qute, compile as is.. adding an expression means that we would need qute processing for all our classes.

Anyway, I really find it handy this way. we may decide to change that at some point, but this will not be in that PR. CC @maxandersen